packages/sites/src/_remove-site-from-index.ts

```diff
@@ -13,21 +13,21 @@ export function _removeSiteFromIndex(
   if (hubRequestOptions.isPortal) {
     return Promise.resolve();
   } else {
-    const url = `${hubRequestOptions.hubApiUrl}/v2/${siteModel.item.id}`;
+    const url = `${getHubApiUrl(hubRequestOptions)}/v2/${siteModel.item.id}`;
```

Comment:
@andrewctate looks like you were the last to edit this, it should probably be `/v3/`, right?

Reply:
Yeah, looks like I missed bumping that API version. Thanks!